frontend/javascripts/oxalis/model/actions/ui_actions.js

```diff
@@ -15,6 +17,11 @@ type SetStoredLayoutsAction = {
   storedLayouts: Object,
 };
 
+type SetBorderOpenStatusAction = {
+  type: "SET_BORDER_OPEN_STATUS",
```

Comment:
Imo I would stick to one term for the borders/sidebars. As the lib named it border, I think we should stick to that so that we do not mix those terms.

Reply:
Fair enough!